libbeat/processors/add_kubernetes_metadata/indexers.go

```diff
@@ -269,7 +269,14 @@ func (h *IPPortIndexer) GetMetadata(pod *Pod) []MetadataIndex {
 	if pod.Status.PodIP == "" {
 		return metadata
 	}
-	for i := 0; i < len(hostPorts); i++ {
+
+	// Add pod IP
+	metadata = append(metadata, MetadataIndex{
+		Index: pod.Status.PodIP,
+		Data:  commonMeta,
+	})
+
+	for i := 1; i < len(hostPorts); i++ {
```

Comment:
Should this be `i := 0`?

Reply:
no, as the first item in the list will be the ip, this filters it to range on ip:port pairs